spec/services/search/postgres/comment_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+RSpec.describe Search::Postgres::Comment, type: :service do
+  let(:comment) { create(:comment) }
+
+  describe "::search_documents" do
+    context "when filtering Commentables" do
+      it "does not include comments from Articles that are unpublished", :aggregate_failures do
+        comment_text = "Ruby on Rails rocks!"
+        published_article = create(:article, title: "Published Article", published: true)
+        unpublished_article = create(:article, title: "Unpublished Article", published: true)
+        comment_on_published_article = create(:comment, body_markdown: comment_text, commentable: published_article)
+        comment_on_unpublished_article = create(:comment, body_markdown: comment_text, commentable: unpublished_article)
+        unpublished_article.update_columns(published: false)
+
+        result = described_class.search_documents(term: "rails")
+        # rubocop:disable Rails/PluckId
+        ids = result.pluck(:id)
+        # rubocop:enable Rails/PluckId
+
+        expect(ids).not_to include(comment_on_unpublished_article.search_id)
+        expect(ids).to include(comment_on_published_article.search_id)
+      end
+
+      # We don't currently search against comments on podcast episodes
```

Comment:
This functionality will likely come back in the future so I figured it didn't hurt to just leave the spec in and skip it 🤷🏽‍♂️

Reply:
I would remove it, we don't know if and how it's going to come back, after all it's not skipped because it fails, it's skipped because the code is not there :)